src/test/java/org/apache/commons/io/IOUtilsTestCase.java

```diff
@@ -1186,6 +1187,42 @@ private void testToString_URL(final String encoding) throws Exception {
         }
     }
 
+    @Test(expected = Test.None.class) public void testResourceToString_ExistingResourceAtRootPackage() throws Exception {
+        IOUtils.resourceToString("/test-file-utf8.bin", Charset.defaultCharset());
+    }
+
+    @Test(expected = Test.None.class) public void testResourceToString_ExistingResourceAtSubPackage() throws Exception {
+        IOUtils.resourceToString("/org/apache/commons/io/FileUtilsTestDataCR.dat", Charset.defaultCharset());
+    }
+
+    @Test(expected = IOException.class) public void testResourceToString_NonExistingResource() throws Exception {
+        IOUtils.resourceToString("/non-existing-file.bin", Charset.defaultCharset());
+    }
+
+    @Test(expected = Test.None.class) public void testResourceToByteArray_ExistingResourceAtRootPackage() throws Exception {
+        IOUtils.resourceToByteArray("/test-file-utf8.bin");
+    }
+
+    @Test(expected = Test.None.class) public void testResourceToByteArray_ExistingResourceAtSubPackage() throws Exception {
+        IOUtils.resourceToByteArray("/org/apache/commons/io/FileUtilsTestDataCR.dat");
+    }
+
+    @Test(expected = IOException.class) public void testResourceToByteArray_NonExistingResource() throws Exception {
+        IOUtils.resourceToByteArray("/non-existing-file.bin");
+    }
+
+    @Test(expected = Test.None.class) public void testResourceToURL_ExistingResourceAtRootPackage() throws Exception {
+        IOUtils.resourceToURL("/test-file-utf8.bin");
+    }
+
+    @Test(expected = Test.None.class) public void testResourceToURL_ExistingResourceAtSubPackage() throws Exception {
+        IOUtils.resourceToURL("/org/apache/commons/io/FileUtilsTestDataCR.dat");
```

Comment:
It is useful in positive test cases to verify things like the size or contents of the result to ensure that it isn't silently returning unuseful results. Ie, assertEquals("Wrong output size", expectedSize, result.size()) or similar as you can find above in this file.